theia-rust-docker/Dockerfile

```diff
@@ -40,7 +40,7 @@ RUN mkdir -p $RUST_HOME && \
     chmod g+rwxs $RUST_HOME 
 RUN umask 0002 && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > /tmp/sh.rustup.rs && sh /tmp/sh.rustup.rs -y
 
-ARG rust_channel=nightly-2019-11-25
+ARG rust_channel=nightly-2020-04-10
```

Comment:
lest-over from an attempt to fix the CI issue with the Rust image? Remove?

Reply:
Yes. With this change it compiles but the tests timeout. Anyway, I would probably just remove the date and leave the nightly channel (the build stops being idempotent but... it would avoid the need to update the date at a later stage and I guess the nightly is stable enough). I also tried to use the stable channel but the nightly is needed due to one dependency of the example project.